android/build.gradle

```diff
@@ -1,16 +1,11 @@
 buildscript {
     repositories {
         google()
-        maven {
-            url 'https://maven.google.com/'
-            name 'Google'
-        }
```

Comment:
The maven repository isn't needed any more - this is what `google()` does

https://developer.android.com/studio/build/dependencies#google-maven